app/views/facilities/_product_list.html.haml

```diff
@@ -10,17 +10,16 @@
     %ul
       - indent_for_icon = products.any? { |p| p.respond_to? :reservations }
       - products.sort.each do |product|
-        %li{ class: product.class.model_name.to_s.downcase }
-          - if local_assigns[:f]
-            = f.fields_for :order_details do |builder|
-              - if session_user.can_override_restrictions?(product) || product.can_be_used_by?(acting_user)
-                = builder.text_field :quantity, value: 0, class: "product_quantity", index: nil
-                = builder.hidden_field :product_id, value: product.id, index: nil
-          = public_calendar_link(product, indent: indent_for_icon)
-          = link_to product.name + (product.is_hidden? ? ' (hidden)' : ''), facility_product_path(current_facility || product.facility, product)
-          - if acting_user.present? && !product.can_be_used_by?(acting_user)
-            %i.fa.fa-lock
-            = " (#{product.class.human_attribute_name(:requires_approval_show)})"
-
-          - if product.offline?
-            = tooltip_icon "fa fa-exclamation-triangle icon-large", t("instruments.offline.note")
+        .product_listing
+          %li{ class: product.class.model_name.to_s.downcase + '_title' }
```

Comment:
This logic was already pretty gnarly, so adding more to it is going to make it even more complex. What would you think of extracting the default behavior out of this file. So we end up with something like this:
```
- products.sort.each do |product|
  - if quicklinks?
    = render "quicklinks", product: product
  - else
    = render "default", product: product
```
That would also allow easier side-by-side comparison of the two views if we can't get them DRYed up.